Tests/CocoaLumberjackTests/DDFileLoggerTests.m

```diff
@@ -106,6 +109,22 @@ - (void)testAutomaticLogFileRollingWhenNotReusingLogFiles {
     XCTAssertEqual(logger.logFileManager.unsortedLogFileInfos.count, 2);
 }
 
+- (void)testAutomaticLogFileRollingWhenNotReusingLogFiles {
+    DDFileLogger *_logger = [[DDFileLogger alloc] initWithLogFileManager:logFileManager];
```

Comment:
The variable is probably better named `newLogger`. By using the underline prefix it might be confused with backing variables.